app/src/services/AdService.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import { addDoc, collection, FirestoreError } from 'firebase/firestore';
+import db from '../firebase/db';
+
+export interface Ad {
+  adDescription: string;
+  adPrice: number;
+  authors: string[];
+  bookDescription: string;
+  bookName: string;
+  courseCodes: string[];
+  tags: string[];
+  publishDate: Date;
+  wear: number;
+  isbn: number;
+}
+
+interface Response {
```

Comment:
export/create a new file for the Response interface

Reply:
Agreed, just need to find a more specific name for the interface since I think Response is used by the fetch API.